src/ui/pages/sett/zcl_abapgit_gui_page_sett_locl.clas.abap

```diff
@@ -279,6 +280,11 @@ CLASS zcl_abapgit_gui_page_sett_locl IMPLEMENTATION.
       iv_readonly = boolc( li_package->are_changes_recorded_in_tr_req( ) = abap_false )
       iv_label    = 'BETA: Enable abapGit flow for this repository (requires transported packages)' ).
 
+    ro_form->textarea(
+      iv_name        = c_id-exclude_remote_paths
+      iv_label       = 'Exclude Remote Paths'
+      iv_hint        = 'List of files patterns (CP operator) to exclude from deserialization' ).
```

Comment:
As you wrote, it's serializing as well

Reply:
"List of files patterns (CP operator) to exclude from syncronization (e.g. unwanted parts of the package, examples...)" ?

Reply:
This plus an example for https://docs.abapgit.org/user-guide/repo-settings/local.html

Reply:
done. Will try to add the docs, but the time is a bit limited